CMakeLists.txt

```diff
@@ -4,104 +4,200 @@
 # Support for special builds (json checking, tests, other stuff)
 
 cmake_minimum_required(VERSION 2.8)
-project(CataclysmDDA)
-set(CMAKE_MODULE_PATH
+
+PROJECT(CataclysmDDA)
+
+SET(CMAKE_MODULE_PATH
     ${CMAKE_MODULE_PATH}
     ${CMAKE_SOURCE_DIR}/cmake
 )
 
 # Build options
-option(LUA      "Lua support (required for some mods)"  OFF)
-option(TILES    "Support for graphical tilesets"        OFF)
-option(SOUND    "Support for in-game sounds & music"    OFF)
-
-# Disable internationalization on Windows and OS X as it is not supported
-# without external libraries
-if (WIN32 OR APPLE)
-    option(LOCALIZE "Support for language localizations" OFF)
-else()
-    option(LOCALIZE "Support for language localizations" ON)
-endif()
-
-# Build settings
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g -Wall -Wextra")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -Wall -Wextra -Werror -Os -DRELEASE")
+option(LUA          "Lua support (required for some mods)"      "OFF")
+option(TILES        "Support for graphical tilesets"            "OFF")
+option(SOUND        "Support for in-game sounds & music"        "OFF")
+option(DEBUG        "Enable debug. Use it for development or bug reporting purposes." "OFF")
+option(ZLEVELS      "Enable experimental z-levels"              "OFF")
+option(USE_HOME_DIR "Use user's home directory for save files." "OFF")
+option(LOCALIZE     "Support for language localizations"        "OFF")
+
+
+MESSAGE("\n * Cataclysm: Dark Days Ahead is a roguelike set in a post-apocalyptic world.")
+MESSAGE("   _________            __                   .__                                ")
+MESSAGE("   \\_   ___ \\ _____   _/  |_ _____     ____  |  |   ___.__   ______  _____      ")
+MESSAGE("   /    \\  \\/ \\__  \\  \\   __\\\\__  \\  _/ ___\\ |  |  <   |  | /  ___/ /     \\     ")
+MESSAGE("   \\     \\____ / __ \\_ |  |   / __ \\_\\  \\___ |  |__ \\___  | \\___ \\ |  Y Y  \\    ")
+MESSAGE("    \\______  /\(____  / |__|  \(____  / \\___  >|____/ / ____|/____  >|__|_|  /    ")
+MESSAGE("           \\/      \\/             \\/      \\/        \\/          \\/       \\/     ")
+MESSAGE("                               --= Dark Days Ahead =--") 
+MESSAGE("\n * http://en.cataclysmdda.com/\n")
+
+MESSAGE(STATUS "${PROJECT} build environment -- \n")
+
+MESSAGE(STATUS "Build realm is   : ${CMAKE_SYSTEM_NAME} ${CMAKE_HOST_SYSTEM_PROCESSOR}\n")
```

Comment:
While this looks really nice with `cmake`, it is slightly annoying when using `ccmake` as I have to confirm this explicitly by pressing a key.

In case someone doesn't known `ccmake`:  it's a curses based UI to cmake, allowing one to change the configuration. It also lists all the available configuration options, which is why I usually use it.

However, when it runs the actual cmake with the proper settings, it captures those messages and required input from the user before it continues. My "standard" process (which works fine for other projects using cmake) is to call `ccmake` fro the build dir, press `c` once to initialize the cmake cache, look at the available options and adjust them, press `c` to configure again, press `g` to generate and exit cmake. The message introduced here requires me to confirm after each `c` and `g`.

But maybe I'm the only one using `ccmake`, and it's only annoying, not a real problem at all.